frame/oracle/src/lib.rs

```diff
@@ -201,67 +202,71 @@ pub mod pallet {
     /// Invalid request ID.
     InvalidRequestId,
     /// Invalid swap request status.
-    InvalidSwapRequestStatus,
-    /// Invalid market maker swap request status.
-    InvalidMarketMakerSwapRequestStatus,
-    /// Market maker buy token does not match swap sell token
-    MarketMakerBuyTokenNotMatchSwapSellToken,
-    /// Market maker has not enough token to sell
-    MarketMakerHasNotEnoughTokenToSell,
+    InvalidRequestSwapStatus,
```

Comment:
```suggestion
    InvalidSwapStatus,
```

Why was this changed? The correct error would be `InvalidSwapRequestStatus` but could also be shortened to `InvalidSwapStatus`.

Reply:
This status is a field for Swap struct rather than a field for a request of confirm_swaps call, that's why `SwapStatus` is clearer as `RequestStatus` mean cause potential confusion here.